whitesource.config

```diff
@@ -31,6 +31,7 @@ includes=**/*.c **/*.cc **/*.cp **/*.cpp **/*.cxx **/*.c++ **/*.h **/*.hpp **/*.
 includes=**/*.tgz **/*.deb **/*.gzip **/*.rpm **/*.tar.bz2
 
 #Exclude file extensions or specific directories by adding **/*.<extension> or **/<excluded_dir>/**
+excludes=**/jenkins/requirements.txt, **/jenkins/pyinstaller/v1/requirements.txt, **/jenkins/pyinstaller/v2/requirements.txt

```

Comment:
Please exclude ha/pcswrap/requirements.txt if that is showing in the errors also

Reply:
As discussed, this file will be excluded if any vulnerability comes up in future because of this